### PR TITLE
Make errors more clear

### PR DESCRIPTION
**Pull request by fregante**
It also looks like Travis isn't working

## Before

```js
❯ remove-github-forks
GithubLib is not a constructor
```

## After

```js
❯ remove-github-forks
TypeError: GithubLib is not a constructor
    at rawGithubFactory (./src/githubFactory.js:10:10)
    at module.exports (./src/githubFactory.js:17:13)
    at Function.api.get (./src/index.js:27:16)
    at Object.<anonymous> (./src/bin.js:56:9)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at /usr/local/lib/node_modules/npm/node_modules/libnpx/index.js:268:14
```

### CODE REVIEW

File: src/bin.js [hunk 1]
```diff
@@ -52,39 +52,43 @@ var assureConfirmation = function(repos, cb) {
...
+        });
       });
-    });
-  }
-);
+    }
+  );
+} catch (error) {
+  console.error(error);
+  process.exit(1);
```
Comment (denis-sokolov) [inline src/bin.js:93]: Let’s reuse existing logic for displaying errors.

```suggestion
  abort(error);
```

File: src/bin.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ if (program.args.length !== 1) program.help();
 var token = program.args[0];
 
 var abort = function(err) {
-  console.error(err.message || err);
+  console.error(err);
```
Comment (denis-sokolov) [inline src/bin.js:33]: Could you hide this behind a `--debug` or other similarly named flag? As much as we benefit from the traceback, it is my believe that for regular users who are not looking for it, a traceback is needlessly loud.

If you haven’t used `commander` before, we need to add an `.option` call after line 14 and check it here like `console.error(program.debug ? err : err.message || err)`.

**Review by denis-sokolov — COMMENTED** [review]
Thanks for this! I have two requests to change, see what you think.

File: src/bin.js (hunk 1 above)
Comment (fregante) [reply src/bin.js]: Definitely!

File: src/bin.js (hunk 2 above)
Comment (fregante) [reply src/bin.js]: I like how easy that is!

**Comment by fregante** [comment]
Thanks for the invite! I'll be opening PRs from branches on this repo soon, if you don't mind, because the Edit button GitHub no longer uses my fork.

**Pull request merged**